static/bidder-info/roulax.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "http://dsp.rcoreads.com/api/{PUBLISHER_PATH}?pid={PID}"
+maintainer:
+  email: "tech@roulax.io"
```

Comment:
This email address is not reachable and does not exists. Please provide a valid email address for communication.